include/libopencm3/stm32/h7/rcc.h

```diff
@@ -0,0 +1,458 @@
+/** @defgroup rcc_defines RCC Defines
+ *
+ * @brief <b>Defined Constants and Types for the STM32H7xx Reset and Clock
+ * Control</b>
+ *
+ * @ingroup STM32H7xx_defines
+ *
+ * @version 1.0.0
+ *
+ * @author Brian Viele <vielster@allocor.tech>
+ *
+ * @date November 2019
+ *
+ * LGPL License Terms @ref lgpl_license
+ *  */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2015 Karl Palsson <karlp@tweak.net.au>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_RCC_H
+#define LIBOPENCM3_RCC_H
+
+#include <libopencm3/stm32/h7/pwr.h>
+
+/**@{*/
+
+/* --- RCC registers ------------------------------------------------------- */
```

Comment:
if you're doing whole new files, doing it with at least basic doxygen grouping from the beginning is certainly preferred.

Reply:
Went ahead and added groups to this file.